<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoubleWindow</class>
 <widget class="QWidget" name="DoubleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_init">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>初始化</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_no1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>串口号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_right4">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>490</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>发送右</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_send1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>41</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_v4">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>532</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>4版常用协议</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_send2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="rate_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>6</number>
   </property>
   <item>
    <property name="text">
     <string>110</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>300</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>product_key</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>240</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_default4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>490</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>00 00 00 03 00 28 03 07 00 20 39 38 32 61 36 38 65 33 61 30 39 61 34 66 62 30 39 34 62 37 65 31 30 34 35 35 63 66 61 64 36 38 00 02 00 01</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_default1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>400</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>00 00 00 03 00 02 03 0b</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_easylink">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>AirLink</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_check1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>校验位：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_left1">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>400</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>发送左</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_stop2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>停止位：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="com_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_default2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>430</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>00 00 00 03 00 02 03 08</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_stop1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>停止位：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rec2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>接收：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_data2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>数据位：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="check_1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Odd</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Even</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Space</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_right2">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>430</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>发送右</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_double">
   <property name="geometry">
    <rect>
     <x>488</x>
     <y>2</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭双串口</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rec1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>接收（F5刷新）：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rate1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>波特率：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_right3">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>460</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>发送右</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_right1">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>400</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>发送右</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_no2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>串口号：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="data_1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_send2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>330</y>
     <width>41</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QComboBox" name="com_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_default3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>00 00 00 03 00 02 03 06</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_rec2">
   <property name="geometry">
    <rect>
     <x>309</x>
     <y>30</y>
     <width>270</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_check2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>校验位：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>-4</y>
     <width>951</width>
     <height>587</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_left2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>430</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>发送左</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rate2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>波特率：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="check_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>300</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Odd</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Even</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Space</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="textEdit_send1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="stop_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_v3">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>532</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>3版常用协议</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_rec1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>270</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_left3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>460</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>发送左</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_data1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>数据位：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ask">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>460</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>询问产品信息</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_left4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>490</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>发送左</string>
   </property>
  </widget>
  <widget class="QComboBox" name="rate_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>6</number>
   </property>
   <item>
    <property name="text">
     <string>110</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>300</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="stop_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="data_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>270</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>2</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>三版换行</string>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>label_init</zorder>
  <zorder>label_no1</zorder>
  <zorder>pushButton_right4</zorder>
  <zorder>pushButton_send1</zorder>
  <zorder>pushButton_open1</zorder>
  <zorder>pushButton_v4</zorder>
  <zorder>textEdit_send2</zorder>
  <zorder>rate_2</zorder>
  <zorder>label_5</zorder>
  <zorder>pushButton_open2</zorder>
  <zorder>lineEdit_default4</zorder>
  <zorder>lineEdit_default1</zorder>
  <zorder>label_easylink</zorder>
  <zorder>label_check1</zorder>
  <zorder>pushButton_left1</zorder>
  <zorder>label_stop2</zorder>
  <zorder>com_2</zorder>
  <zorder>lineEdit_default2</zorder>
  <zorder>label_stop1</zorder>
  <zorder>label_rec2</zorder>
  <zorder>label_data2</zorder>
  <zorder>check_1</zorder>
  <zorder>pushButton_right2</zorder>
  <zorder>pushButton_double</zorder>
  <zorder>label_rec1</zorder>
  <zorder>label_rate1</zorder>
  <zorder>pushButton_right3</zorder>
  <zorder>pushButton_right1</zorder>
  <zorder>label_no2</zorder>
  <zorder>data_1</zorder>
  <zorder>pushButton_send2</zorder>
  <zorder>com_1</zorder>
  <zorder>lineEdit_default3</zorder>
  <zorder>textEdit_rec2</zorder>
  <zorder>label_check2</zorder>
  <zorder>pushButton_left2</zorder>
  <zorder>label_rate2</zorder>
  <zorder>check_2</zorder>
  <zorder>textEdit_send1</zorder>
  <zorder>stop_1</zorder>
  <zorder>pushButton_v3</zorder>
  <zorder>textEdit_rec1</zorder>
  <zorder>pushButton_left3</zorder>
  <zorder>label_data1</zorder>
  <zorder>label_ask</zorder>
  <zorder>pushButton_left4</zorder>
  <zorder>rate_1</zorder>
  <zorder>stop_2</zorder>
  <zorder>data_2</zorder>
  <zorder>checkBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
